pkg/git/git.go

```diff
@@ -0,0 +1,53 @@
+package git
```

Comment:
This package doesn't appear to have any tests? I assume you are writing them? :D

Reply:
Oh I see the resolved conversation. Sorry.